webpack/stripes-error-logging-plugin.js

```diff
@@ -0,0 +1,27 @@
+// This webpack plugin generates a virtual module containing the
+// error-handler configuration.
+
+const logger = require('./logger')('stripesErrorLoggingPlugin');
+
+module.exports = class StripesErrorLoggingPlugin {
+  constructor(options) {
+    logger.log('initializing...');
+
+    const defaultErrorLogging = {
+      rollbar: null,
+      bugsnag: null,
+      sentry: null,
+    };
```

Comment:
You know, aside from this assignment, which should just be `const defaultErrorLogging = {}`, this PR wasn't all bad!